source/templates/partials/_modal-test.twig

```diff
@@ -15,14 +15,30 @@
 
 <div class="form__wrapper">
   <label class="form__label" for=""></label>
-  <input class="form__field" id="" type="date" name="" value="">
+  <input class="form__field" id="" type="text" name="" value="" aria-describedby="calendar">
+  <p id="calendar"></p>
```

Comment:
- We can format ID for help message like that "help-<id-input>" (like that we are sure this help message is unique and we understand directly for which input it is for). For example `help-date` or `help-age`